prboom2/.editorconfig

```diff
@@ -0,0 +1,10 @@
+# top-most EditorConfig file
```

Comment:
Not sure if this is intended to be here, but if it is then let's make that a separate PR so it doesn't add noise for anyone trying to see just the relevant changes.